hardware_interface/source/eps.c

```diff
@@ -53,16 +56,28 @@ SAT_returnState eps_refresh_instantaneous_telemetry() {
     eps_instantaneous_telemetry_t telembuf;
     int res = csp_ping(EPS_APP_ID, 10000, 100, CSP_O_NONE);
 
-    csp_transaction_w_opts(CSP_PRIO_LOW, EPS_APP_ID, EPS_INSTANTANEOUS_TELEMETRY,
-                           10000, &cmd, sizeof(cmd), &telembuf,
-                           sizeof(eps_instantaneous_telemetry_t), CSP_O_CRC32);
+    csp_transaction_w_opts(CSP_PRIO_LOW, EPS_APP_ID, EPS_INSTANTANEOUS_TELEMETRY, 10000, &cmd, sizeof(cmd),

```

Comment:
Since we're here, could you replace these `10000` values with EPS_REQUEST_TIMEOUT wherever it appears (I see that EPS_REQUEST_TIMEOUT is smaller, but it should be plenty long)

Reply:
sure thing